Gems/Atom/Feature/Common/Assets/Shaders/Math/GaussianFilterFloatVertical.azsl

```diff
@@ -1,60 +0,0 @@
-/*
- * Copyright (c) Contributors to the Open 3D Engine Project.
- * For complete copyright and license terms please see the LICENSE at the root of this distribution.
- *
- * SPDX-License-Identifier: Apache-2.0 OR MIT
- *
- */
-
-// [GFX TODO][ATOM-3365] optimization using intermediary results in groupshared memory.
-
-#include <Atom/Features/Math/Filter.azsli>
-#include <Atom/Features/Math/FilterPassSrg.azsli>
-#include <Atom/Features/Shadow/ShadowmapAtlasLib.azsli>
-
-[numthreads(16,16,1)]
-void MainCS(uint3 dispatchId: SV_DispatchThreadID)
-{
-    const float3 inputSize = GetImageSize(FilterPassSrg::m_inputImage);
-    const float3 outputSize = GetImageSize(FilterPassSrg::m_outputImage);
-
-    const uint shadowmapIndex = GetShadowmapIndex(
-        FilterPassSrg::m_shadowmapIndexTable,
-        dispatchId,
-        inputSize.x);
-    // Early return if thread is outside of shadowmaps.
```

Comment:
It looks like the kawase blur doesn't have any of these early returns, even though it's using the same GetShadowmapIndex function. Is there a reason that's not necessary anymore?

Reply:
Really good eye, thanks for noticing this! I've added the old checks back in.